app/assets/stylesheets/complaints.scss

```diff
@@ -2,6 +2,7 @@
   background-color: #fff;
   border: 2px solid #dfe1e2;
   border-radius: 5px;
+  padding-bottom: 20px;
```

Comment:
[suggestion] I think we've been mostly keeping spacing using the `rem` unit and it would be good to continue that.

Reply:
I added something in a recent commit that may be of help, or it might just complicate matters depending on what you're aiming for:

https://github.com/OHS-Hosting-Infrastructure/complaint-tracker/blob/main/app/assets/stylesheets/complaints.scss#L5